static/templates/stream_settings/stream_settings_overlay.hbs

```diff
@@ -26,6 +26,10 @@
                         <i class="fa fa-remove" aria-hidden="true"></i>
                     </button>
                 </div>
+                <div class="nothing-unsubscribed">
+                    <span>No streams to show</span>
```

Comment:
All user-facing strings should be tagged for translation. 
Check out https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.